workbench-core/datasets/src/ddbDataSetMetadataPlugin.ts

```diff
@@ -197,4 +201,13 @@ export class DdbDataSetMetadataPlugin implements DataSetMetadataPlugin {
 
     return dataSet.id!;
   }
+
+  private async _removeDataSetFromDdb(dataSetId: string): Promise<void> {
+    const dataSetKey = {
+      pk: `${this._dataSetKeyType}#${dataSetId}`,
```

Comment:
reason why pk and sk are the same?

Reply:
This is to distinguish the dataset metadata (pk: DATASET, sk: DATASET) from the dataset's endpoint metadata (pk: DATASET, sk: ENDPOINT)